diagnostic.md

```diff
@@ -33,7 +40,7 @@ y = 10
 After each line of code executes, what are the values of x and y?  Do the variables change?  Do their values?
 
 <!-- solution below -->
-
+Once the code is executed, x equals 3 and y equals 10. The variables x and y themselves don't change but their values do.
```

Comment:
Right, we also wanted you to list the initial values of `x` and `y` but I think you probably knew what they were.